src/guc.c

```diff
@@ -473,7 +466,7 @@ _guc_init(void)
 							"Maximum cached chunks",
 							"Maximum number of chunks stored in the cache",
 							&ts_guc_max_cached_chunks_per_hypertable,
-							100,
+							1024,
```

Comment:
we seem to be caching chunks for  [up-to-16 hypertables](https://github.com/timescale/timescaledb/blob/caf79e0f5e3b789f70361f4fd3093afcea3a1da0/src/hypertable_cache.c#L68) at a time..

so if the above 25K per entry applies here as well - this cache will probably take up at tops `25K*1024*16 ~ 406MB` with the current setting - I'm not sure if that should be allowed in the default configuration

Reply:
If a session touches 16 hypertables with 1024 chunks with a modest size of 1 GB each, it is processing 16 TB of data. I think it is reasonable to use some 400 MB for metadata in this case.